src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,75 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.util.ArrayList;
+public class TaskList {
+    private ArrayList<Task> list = new ArrayList<>();
+
+    public TaskList(ArrayList<Task> list) {
+        this.list = list;
+    }
+
+    public void addTask(Task task) {
+        if (task instanceof Todo) {
```

Comment:
We might want to avoid using `instanceof`. Is there a better way to check the type of Task? (Applies to other places where `instanceof` is used)